Board/src/main/java/com/example/board/post/JPAPostService.java

```diff
@@ -0,0 +1,74 @@
+package com.example.board.post;
+
+import com.example.board.member.JPAMemberService;
+import com.example.board.member.MemberJPARepository;
+import javassist.NotFoundException;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class JPAPostService implements PostService{
+
+    PostJPARepository repository;
+    PostConverter converter;
+    MemberJPARepository memberRepository;
+
+    public JPAPostService(PostJPARepository repository, PostConverter converter, MemberJPARepository memberRepository) {
+        this.repository = repository;
+        this.converter = converter;
+        this.memberRepository = memberRepository;
+    }
+
+    @Transactional
+    @Override
+    public PostDto createPost(PostDto postDto) {
+        Post post = PostConverter.of(postDto);
+        repository.save(post);
+        post.setMember(memberRepository.findByName(postDto.memberName()).get());
```

Comment:
```suggestion
        member = memberRepository.findByName(postDto.getName())
        Post post = repository Post(..., member)
        repository.save(post);
```
이런식으로는 어려울까요? setter를 써야할 이유를 못느끼겠네요